examples/samples/README.md

```diff
@@ -2,42 +2,40 @@
 
 ## Building the library
 - Clone the repo and cd to the repo directory
-```
+```bash
 git clone https://github.com/azure/azure-service-bus-node.git
 cd azure-service-bus-node
 ```
 - Install typescript, ts-node globally (optional, but very useful)
-```
+```bash
 npm i -g typescript
 npm i -g ts-node
 ```
 - NPM install from the root of the package
-```
+```bash
 npm i
 ```
 - Build the project
-```
+```bash
 npm run build
 ```
 
 ## Before executing a sample
-- Go to the Azure Portal(https://portal.azure.com)
-- Here are the docs which would help you create a service bus resource in the portal : https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-nodejs-how-to-use-queues
-```
-Go to Dashboard > Service Bus > <your-servicebus-namespace> 
-```
+- Go to the [Azure Portal](https://portal.azure.com)
+- Here are the docs which would help you create a service bus resource in the portal : [ServiceBus - NodeJS DOCS](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-nodejs-how-to-use-queues)
+- In the portal, go to **Dashboard > Service Bus > _your-servicebus-namespace_**
 - If you'd like to work with Queues, find the "Queues" tab right under "Entities" at <your-servicebus-namespace> and create a Queue.
-- Go to the Shared access policies under settings tab and add a new one. 
+- Go to the **Shared access policies** under **Settings** tab and add a new one. 
```

Comment:
I dont recall doing this step. I used the default one. Do we really need to do this?

Reply:
I think you're referring **RootManageSharedAccessKey** as the **default one**. Yes, it is automatically created for the namespace and has permissions for the entire namespace. 

In the [Docs](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas), it is recommended to treat this rule like an administrative root account and don't use it in the application. 

The one I mentioned in the README, iterates the steps to create the Access Keys exclusive to the specific created Queue.

Reply:
Yes, I was referring to **RootManageSharedAccessKey**.

For the purpose of reducing the number of things a user has to do to run the samples, I would suggest to not have this step as mandatory. This would now need the user to know what a shared access policy is and how to create one.

Here, lets use the **RootManageSharedAccessKey** itself. You can add a side note that this one is like an admin root account and if the user wants to use restricted access they can refer to the doc on using [Shared Access Signatures](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas)